components/borrow.tsx

```diff
@@ -48,56 +51,55 @@ const EditSvg = (props: SVGProps<any>) => (
 )
 
 const Position = ({
-  currency,
-  apy,
-  borrowed,
-  collateral,
-  collateralSymbol,
-  expiry,
+  position,
   price,
   collateralPrice,
-  ltv,
-  liquidationThreshold,
   onRepay,
   onBorrowMore,
   onEditCollateral,
   onEditExpiry,
   ...props
 }: {
-  currency: Currency
-  apy: string
-  borrowed: string
-  collateral: string
-  collateralSymbol: string
-  expiry: string
-  price: string
-  collateralPrice: string
-  ltv: string
-  liquidationThreshold: string
+  position: LoanPosition
+  price: number
+  collateralPrice: number
   onRepay: () => void
   onBorrowMore: () => void
   onEditCollateral: () => void
   onEditExpiry: () => void
 } & React.HTMLAttributes<HTMLDivElement>) => {
-  collateralPrice // TODO remove
+  const currentTimestamp = Math.floor(new Date().getTime() / 1000)
   return (
     <div className="rounded-xl shadow bg-gray-50 dark:bg-gray-900" {...props}>
       <div className="flex justify-between rounded-t-xl p-4 bg-white dark:bg-gray-800">
         <div className="flex items-center gap-3">
           <img
-            src={getLogo(currency)}
-            alt={currency.name}
+            src={getLogo(position.underlying)}
+            alt={position.underlying.name}
             className="w-8 h-8"
           />
           <div className="flex flex-col">
-            <div className="font-bold">{currency.symbol}</div>
-            <div className="text-gray-500 text-sm">{currency.name}</div>
+            <div className="font-bold">{position.underlying.symbol}</div>
+            <div className="text-gray-500 text-sm">
+              {position.underlying.name}
+            </div>
           </div>
         </div>
         <div className="flex flex-col items-end">
-          <div className="font-bold">{apy}</div>
+          <div className="font-bold">
+            {calculateApr(
+              Number(position.interest) / Number(position.amount),
+              position.expiryTimestamp - currentTimestamp,
```

Comment:
You must use the timestamp the borrow was initiated, or the APR won't be fixed.

Reply:
Could you also check if this one is correct? : https://github.com/clober-dex/coupon.finance/blob/dev/components/deposit.tsx#L55

Reply:
Wow they're both wrong lol

Reply:
They're both wrong. Once a position is made the apr, apy does not change

Reply:
https://github.com/clober-dex/coupon.finance/pull/77/commits/315f0be921174528ad374363e9636ea6faa5f2de

@onemonster Could you check this commit?